pkg/controller/tidbcluster/tidb_cluster_controller.go

```diff
@@ -76,11 +76,11 @@ func NewController(
 	tikvFailoverPeriod time.Duration,
 	tidbFailoverPeriod time.Duration,
 ) *Controller {
-	eventBroadcaster := record.NewBroadcaster()
-	eventBroadcaster.StartLogging(glog.Infof)
+	eventBroadcaster := record.NewBroadcasterWithCorrelatorOptions(record.CorrelatorOptions{QPS: 10})
+	eventBroadcaster.StartLogging(glog.V(4).Infof)
```

Comment:
`glog.Info` is too noisy.

Reply:
events are often the most important messages that end users care, I'd prefer to have it on in logs by default, how about `v(2)`?
If the logging of the event is too noisy, we should reduce the rate of events we reported to the API server. 
Another reason is the event can be dropped, in that case, we can find the full events from the logs.

Reply:
reasonable